app/utils/ResourceActions.js

```diff
@@ -22,7 +24,7 @@ export const getResourceActions = resource => ({
   phone: {
     name: 'Call',
     icon: 'call',
-    link: `tel:${resource.phones[0].number}`,
+    link: `tel:${_.get(resource, 'phones[0].number')}`,
```

Comment:
Should we not display this button at all if the resource doesn't have a phone number?

Reply:
Good call! Updated